easybuild/tools/options.py

```diff
@@ -93,7 +93,7 @@ def basic_options(self):
             'dry-run-short': ("Print build overview incl. dependencies (short paths)", None, 'store_true', False, 'D'),
             'force': ("Force to rebuild software even if it's already installed (i.e. if it can be found as module)",
                       None, 'store_true', False, 'f'),
-            'job': ("Submit the build as a job", None, 'store_true', False),
+            'job': ("Submit the build as a job", 'choice', 'store_or_None', 'pbs', ['pbs']),
```

Comment:
Just talked to @boegel about this: I'll split the option into a boolean `--job` and a choice `--job-backend` option.